src/include/duckdb/main/settings.hpp

```diff
@@ -496,6 +496,15 @@ struct ExportLargeBufferArrow {
 	static Value GetSetting(ClientContext &context);
 };
 
+struct ProduceArrowStringView {
+	static constexpr const char *Name = "produce_arrow_string_view";
+	static constexpr const char *Description = "If strings should be produced by DuckDB as string_views";
```

Comment:
This looks like it could cause conflation with std::string_view. Since they're not related, could we use:
```suggestion
	static constexpr const char *Description = "If strings should be produced by DuckDB in Utf8View format instead of Utf8";
```